creator-node/src/snapbackSM.js

```diff
@@ -1,14 +1,12 @@
 const Bull = require('bull')
 const axios = require('axios')
+
 const utils = require('./utils')
 const models = require('./models')
 const { logger } = require('./logging')
 
-// For local dev, configure this to be the interval when SnapbackSM is fired
-const DevDelayInMS = 3000
-
 // Represents the maximum number of syncs that can be issued at once
-const MaxParallelSyncJobs = 10
+const MaxParallelSyncJobs = 5
```

Comment:
does this throttle the number of manual operations that can be processed at once? previously it was up to 10 simultaneously vs. now at 5. I see the thought process here (10 across 2 queues so 5 each) just want to confirm this value makes sense

Reply:
yeah my concern was that we are already seeing high memory usage on node and increasing total max conc to 15 seemed risky. maybe we can up it later if we think we have room?